Tools/TestWebKitAPI/Tests/WebKitCocoa/ScreenTime.mm

```diff
@@ -546,6 +550,42 @@ - (void)userContentController:(WKUserContentController *)userContentController d
 
 #endif
 
+TEST(ScreenTime, FetchData)
+{
+    if (![PAL::getSTWebHistoryClass() instancesRespondToSelector:@selector(fetchAllHistoryWithCompletionHandler:)])
+        return;
+
+    __block bool calledScreenTimeFetchAllHistory = false;
+    InstanceMethodSwizzler swizzler {
+        PAL::getSTWebHistoryClass(),
+        @selector(fetchAllHistoryWithCompletionHandler:),
+        imp_implementationWithBlock(^(id object, NSSet<NSURL *> *urls, NSError *error) {
```

Comment:
Couple things:
1. This method signature is incorrect, since `fetchAllHistoryWithCompletionHandler:` take a single completion handler parameter. Not a set and an error parameter.
2. After fixing that, we can call the completion handler with a mock set of URLs to achieve better testing.